### PR TITLE
add typespec-ts configs in all the tspconfig

### PR DESCRIPTION
**Pull request by kazrael2119**
fix https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4132078&view=logs&j=47092895-c7c8-567a-c7a6-8c88f3ec178e&t=d293a8e8-c6af-5091-cdf7-4c66482dd6f2&l=111427

service which has released before:
```
specification\deviceregistry\DeviceRegistry.Management\tspconfig.yaml
specification\devopsinfrastructure\Microsoft.DevOpsInfrastructure.Management\tspconfig.yaml
specification\edgezones\EdgeZones.Management\tspconfig.yaml
specification\fabric\Microsoft.Fabric.Management\tspconfig.yaml
specification\informatica\Informatica.DataManagement.Management\tspconfig.yaml
specification\mongocluster\DocumentDB.MongoCluster.Management\tspconfig.yaml
specification\scvmm\ScVmm.Management\tspconfig.yaml
```

need add service-dir:(updated)
```
specification\hybridconnectivity\PublicCloud.Management\tspconfig.yaml
specification\liftrastronomer\Astronomer.Astro.Management\tspconfig.yaml
specification\liftrqumulo\Qumulo.Storage.Management\tspconfig.yaml
specification\loadtestservice\LoadTestService.Management\tspconfig.yaml
specification\monitor\Microsoft.Monitor.Management\tspconfig.yaml
specification\networkanalytics\NetworkAnalytics.Management\tspconfig.yaml
specification\playwrighttesting\PlaywrightTesting.Management\tspconfig.yaml
specification\servicenetworking\ServiceNetworking.Management\tspconfig.yaml
specification\sphere\Sphere.Management\tspconfig.yaml
specification\vmware\Microsoft.AVS.Management\tspconfig.yaml
specification\workloads\Workloads.SAPVirtualInstance.Management\tspconfig.yaml
specification\workloads\Workloads.SAPDiscoverySite.Management\tspconfig.yaml
```

not sure the correct data-plane package name:
```
specification\eventgrid\Azure.Messaging.EventGrid.SystemEvents\tspconfig.yaml(updated)
specification\eventgrid\Azure.Messaging.EventGrid\tspconfig.yaml(updated)
specification\machinelearningservices\AzureAI.Assets\tspconfig.yaml(updated)
specification\playwrighttesting\PlaywrightTesting\tspconfig.yaml(skipped)
specification\programmableconnectivity\Azure.ProgrammableConnectivity\tspconfig.yaml(updated)
specification\quantum\Quantum.Workspace\tspconfig.yaml(skipped)
specification\voiceservices\VoiceServices.Provisioning\tspconfig.yaml(skipped)
```

new mgmt-plane package name and they don't have readme.typescript.md:
```
specification\iotoperations\IoTOperations.Management\tspconfig.yaml
specification\iotoperationsdataprocessor\IoTOperationsDataProcessor.Management\tspconfig.yaml(skipped)
specification\iotoperationsmq\IoTOperationsMQ.Management\tspconfig.yaml(skipped)
specification\iotoperationsorchestrator\IoTOperationsOrchestrator.Management\tspconfig.yaml(skipped)
specification\portalservices\CopilotSettings.Management\tspconfig.yaml(skipped)
specification\portalservices\Extension.Management\tspconfig.yaml(skipped)
specification\purviewpolicy\PurviewPolicy.Management\tspconfig.yaml(skipped)
specification\splitio\SplitIO.Experimentation.Management\tspconfig.yaml(skipped)
```

new mgmt-plane packages and they have readme.typescript.md:
```
specification\kubernetesruntime\KubernetesRuntime.Management\tspconfig.yaml(updated)
specification\mobilepacketcore\MobilePacketCore.Management\tspconfig.yaml(updated)
specification\mpcnetworkfunction\MpcNetworkFunction.Management\tspconfig.yaml(updated)
specification\portal\Dashboard.Management\tspconfig.yaml(hold on)
specification\portal\TenantConfiguration.Management\tspconfig.yaml(hold on)
specification\verifiedid\Microsoft.VerifiedId.Management\tspconfig.yaml(updated)
specification\workloads\Workloads.Operations.Management\tspconfig.yaml(skipped)
specification\workloads\Workloads.SAPMonitor.Management\tspconfig.yaml(updated)
```

### CODE REVIEW

File: specification/cosmos-db/resource-manager/readme.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ These are the global settings for the Cosmos-DB API.
 ``` yaml
 title: CosmosDBManagementClient
 openapi-type: arm
-tag: package-preview-2024-09
+tag: package-2024-08
```
Comment (kazrael2119) [inline specification/cosmos-db/resource-manager/readme.md:30]: need revert this

**Review by qiaozha — APPROVED** [review]

**Comment by wanlwanl** [comment]
Found issue https://github.com/Azure/autorest.typescript/issues/2836

**Review by msyyc — APPROVED** [review]

**Comment by mentat9** [comment]
@kazrael2119 - Please fix conflicts so the validation pipeline can run.

File: specification/informatica/Informatica.DataManagement.Management/tspconfig.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 parameters:
   "service-dir":
-    default: "sdk/informaticadatamanagement"
+    default: "sdk/informatica"
```
Comment (MaryGao) [inline specification/informatica/Informatica.DataManagement.Management/tspconfig.yaml:3]: @ziwei why do we have this change?

File: specification/kubernetesruntime/KubernetesRuntime.Management/tspconfig.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 parameters:
   "service-dir":
-    default: "sdk/containerorchestratorruntime"
+    default: "sdk/kubernetesruntime"
```
Comment (MaryGao) [inline specification/kubernetesruntime/KubernetesRuntime.Management/tspconfig.yaml:3]: also this one?

**Pull request merged**